base/replutil.jl

```diff
@@ -173,6 +173,24 @@ end
     showerror(io, e)
 
 Show a descriptive representation of an exception object.
+
+# Examples
+```jldoctest
+julia> struct MyException <: Exception
+           msg::AbstractString
+       end
+
+julia> function Base.showerror(io::IO, err::MyException)
+           print(io, "MyException: ")
+           print(io, err.msg)
+       end
+
+julia> err = MyException("test exception")
```

Comment:
Can we do
```julia
julia> throw(MyException("test exception"))
ERROR: MyException: test exception
```
instead? Somewhat more "realistic" case.

Reply:
why not both? i'll push an update (it may not be clear to newbies that `throw` will end up calling `showerror`)

Reply:
Right, but in normal code you will never call `showerror` yourself, so its a bit misleading.